src/ssis_migrator/parsers/dtsx_parser.py

```
@@ -0,0 +1,445 @@
+"""
+DTSX Parser - Main parser for SSIS .dtsx files
+"""
+
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from typing import Dict, List, Any, Optional
+from dataclasses import dataclass, field
+from ..core.logger import LoggerMixin
+from ..core.error_handler import (
+    ErrorHandler, ParsingError, FileSystemError, create_error_context,
+    ErrorSeverity, ErrorCategory
```

Ruff (0.11.9)

`..core.error_handler.ErrorCategory` imported but unused

Remove unused import: `..core.error_handler.ErrorCategory`

(F401)